tools/calculate_ref_alt_counts.py

```diff
@@ -33,6 +33,56 @@ def get_read_length(bam_cache, sample_cap=20):
 
 
 # Requires BAM files and events list
+def calculate_ref_count(bpp, read_length, reference_genome, bam_cache, buffer=1):
+    """
+    Attempt to calculate the amount of reference reads that support the bpp based on position
```

Comment:
could you also expand a bit here on what this functions is trying to do or rather how. Like "Creates expected ref/alt sequences based on the breakpoint pair and checks for an exact substring match in reads"

Reply:
updated docstring